### DISCUSSION TITLE
i3wm freeze issue

### DISCUSSION THREAD

**Discussion by haoxianhan** [i3/i3#4970] [Q&A]
I sometimes got workspace freezing, I wonder if application(Alacritty) with gpu cause it.
I use thinkpad x1c 6 gen, it should well understand by video :https://haoxian.tech/tmp/i3wm_issue.html. In video, workspace 1 and 3 worked fine but workspace 2 did not.
btw, my issue is similar to https://forum.archlabslinux.com/t/i3-gaps-window-freeze/3389

**Comment by Airblader** [comment]
This is probably an issue with the compositor; try disabling it and see if the issue persists.

**Reply by haoxianhan** [reply]
thx, it seems that compositor causes this problem.
I use `picom`, after killed the proc everything works fine, and I found doc said "To autostart picom as a background process for a session, the -b argument can be used (may cause a display freeze)", I would try start as background job with `&` instead of  `-b`.

**Reply by Airblader** [reply]
That won't do much, I think. Make sure your video drivers are up to date, and experiment with picom settings, i.e. different render backends or disabling individual features like fading.
